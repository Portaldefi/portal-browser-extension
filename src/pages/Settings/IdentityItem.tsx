import React from 'react';
import { Checkbox, Grid, Header } from 'semantic-ui-react';

interface IIdentityItem {
  name: string,
  comment: string
};

export default ({ name, comment }: IIdentityItem) => {
  return (
    <Grid className='p-16 py-9 bg-white'>
      <Grid.Column width='eleven' textAlign='left'>
        <Header as='p' className='description'>{name}</Header>
        <Header as='p' className='comment'>{comment}</Header>
      </Grid.Column>
    </Grid>
  )
}