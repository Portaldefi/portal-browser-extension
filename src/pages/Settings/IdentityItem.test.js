import IdentityItem from './IdentityItem';
import { create, act } from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';

describe('IdentityItemComponent', () => {
  it('should work', () => {
    let tree;
    act(() => {
      tree = create(
        <BrowserRouter>
          <IdentityItem />
        </BrowserRouter>
      );
    });

    expect(tree).toMatchSnapshot();
  });
  afterAll(() => jest.resetModules());
});