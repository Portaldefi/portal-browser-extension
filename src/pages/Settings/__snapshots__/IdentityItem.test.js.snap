// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IdentityItemComponent should work 1`] = `
<div
  className="ui grid p-16 py-9 bg-white"
>
  <div
    className="left aligned eleven wide column"
  >
    <p
      className="ui header description"
    />
    <p
      className="ui header comment"
    />
  </div>
</div>
`;
